pkg/database/sql/reader.go

```diff
@@ -52,6 +55,13 @@ func (db *sqlDatabaseImpl) QueryResources(ctx context.Context, kind, apiVersion,
 		if labelFilters.NotIn != nil {
 			sb.Where(db.filter.NotInLabelFilter(sb.Cond, labelFilters.NotIn, mainWhereClause))
 		}
+		if fieldReqs != nil {
+			slog.Info("Processing field requirements", "count", len(fieldReqs))
```

Comment:
I don't think this log is helpful, just seeing the count makes you want to see the actual items. Also, make it debug if you want to keep it.